docs/_includes/toc-page.html

```diff
@@ -0,0 +1,73 @@
+<div class="page-wrapper" tabindex="-1" role="main">
+    {% if page.cover %}
+    <img src="{{ page.cover }}"
+         width="100%"
+         height="{{ page.cover_height | default: '100%' }}"
+         alt="{{ page.title | escape }}"
+         style="object-fit: cover;"
+    />
+    {% endif %}
+
+    <div class="page-inner">
+        <div id="book-search-results">
+            <div class="search-noresults">
+                <section class="normal markdown-section">
+                    <h1>VMware Secrets Manager Blog</h1>
+
+                    {% for collection in site.collections %}
+                        {% if collection.output %}
+                            {% if collection.label == "posts" %}
+                                {% assign reversed_posts = site[collection.label] | reverse %}
+                            {% else %}
+                                {% assign reversed_posts = site[collection.label] %}
+                            {% endif %}
+
+                        {% for post in reversed_posts %}
+
+
+                        {% if post.kind == "VSecMBlog" %}
```

Comment:
I defined a new attribute "kind" on the posts.
so if the page is of kind VSecMBlog, then we display its details in this listing.